app/docs/dev/security/api-server-auth.md

```diff
@@ -64,7 +64,7 @@ This group is [authorized by default](/docs/{{ page.version }}/security/api-acce
 
    Execute the following command on the machine where you deployed the control plane.
    ```sh
-   curl http://localhost:5681/global-secrets/admin-user-token | jq -r .data | base64 -d
+   wget -q -O - http://localhost:5681/global-secrets/admin-user-token | grep data | cut -d " " -f 3 | tr -d '"' | base64 -d
```

Comment:
I had a version of this with both curl and wget but it seems broken :/ I'll create an issue.

Reply:
The issue is described here - https://github.com/kumahq/kuma-website/issues/1127

Reply:
Why not `jq`?

Reply:
There is no `jq` in the `kuma-cp` image.

Reply:
Can we just execute the `wget` line using `docker exec` and pipe it into `jq` on the user's machine and include that in the docker installation page? These instructions are more general than just docker installation.

Reply:
Yeah, that will also work. It's just that `wget` is more commonly pre-installed and I did not think that would break anything here.